app/controllers/comments_controller.rb

```diff
@@ -5,6 +5,8 @@ class CommentsController < ApplicationController
 
   def create
     @comment = @curiosity.comments.new(comment_params)
+    authorize @comment
```

Comment:
I think `authorize` should be on a `before_action` instead of calling them on each controller action